client/src/components/BoulderCard.spec.jsx

```diff
@@ -22,23 +24,24 @@ const Boulder = {
 describe('BoulderCard', () => {
   it('renders the a list item with a title, an img, 5 details and a button and renders four more details after button click', () => {
     render(
+      <Router>
       <BoulderCard
         id={Boulder.number}
         name={Boulder.name}
         sector={Boulder.sector}
         level={Boulder.level}
-        handle_color={Boulder.handle_color}
+        hold_color={Boulder.handle_color}
```

Comment:
If you do the destructuring as suggested above in the BoulderCard.jsx, you could just pass "Boulder" as a prop here.

Reply:
Done, thank you!